ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py

```diff
@@ -640,3 +640,43 @@ def test_numpy_fmod(
         dtype=dtype,
         subok=True,
     )
+
+
+# modf
+@handle_frontend_test(
+    fn_tree="numpy.modf",
+    dtypes_values_casting=np_frontend_helpers.dtypes_values_casting_dtype(
+        arr_func=[
+            lambda: helpers.dtype_and_values(
+                available_dtypes=helpers.get_dtypes("numeric"),
+                num_arrays=1,
+                max_dims=1,
```

Comment:
Some of these arguments aren't supported you might have wrong-typed them kindly check the helpers documentation to know accepted arguments https://lets-unify.ai/ivy/testing/hypothesis_helpers.html

Reply:
Still the following data generation is causing failures would you check on them